Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -177,8 +177,9 @@ extension MongoCollection {
      * Creates an index over the collection for the provided keys with the provided options.
      *
      * - Parameters:
-     *   - model: An `IndexModel` representing the keys and options for the index
-     *   - writeConcern: Optional WriteConcern to use for the command
+     *   - keys: a `Document` specifing the keys for the index
+     *   - options: Optional `IndexOptions` to use for the index
+     *   - options: Optional `CreateIndexOptions` to use for the command
```

Comment:
Looks like `options` should not be listed twice here.

Reply:
good catch, the second parameter is named `commandOptions`.